src/sider/dialogmoter/motebehov/MotebehovKvittering.tsx

```diff
@@ -140,62 +169,170 @@ export function MotebehovArbeidsgiverKvittering({
+  motebehov,
+  oppfolgingstilfelle,
+  skjemaType,
+}: {
+  motebehov: MotebehovVeilederDTO | undefined;
+  oppfolgingstilfelle: OppfolgingstilfelleDTO;
+  skjemaType: MotebehovSkjemaType | null;
+}) {
+  const { currentLedere } = useLedereQuery();
+  const ledereInCurrentTilfelle =
+    activeNarmesteLederForCurrentOppfolgingstilfelle(
+      currentLedere,
+      oppfolgingstilfelle
+    );
+
+  return motebehov ? (
+    <MotebehovArbeidsgiverKvittering
+      motebehov={motebehov}
+      skjemaType={motebehov?.skjemaType ?? null}
+    />
+  ) : (
+    <MotebehovKvitteringInnholdArbeidsgiverUtenMotebehov
+      ledereUtenInnsendtMotebehov={ledereInCurrentTilfelle}
+      skjemaType={skjemaType}
+    />
+  );
+}
+
+function UbehandledeMotebehovUtenforTilfelle({
+  sorterteMotebehovUtenforTilfelle,
+}: {
+  sorterteMotebehovUtenforTilfelle: MotebehovVeilederDTO[];
+}) {
+  const ubehandledeEldreMotebehov = getUbehandletSvarOgMeldtBehov(
+    sorterteMotebehovUtenforTilfelle
+  );
+  const motebehovArbeidstaker = ubehandledeEldreMotebehov.find(
+    isArbeidstakerMotebehov
+  );
+  const motebehovArbeidsgiver = ubehandledeEldreMotebehov.find(
+    (motebehov) => !isArbeidstakerMotebehov(motebehov)
+  );
+
+  return motebehovArbeidstaker || motebehovArbeidsgiver ? (
+    <div className="flex flex-col gap-2">
+      <BodyShort size="small">
+        Ubehandlede møtebehov fra tidligere oppfølgingstilfelle:
+      </BodyShort>
+      {motebehovArbeidstaker && (
+        <MotebehovKvitteringInnholdArbeidstaker
+          arbeidstakersMotebehov={motebehovArbeidstaker}
+          skjemaType={motebehovArbeidstaker?.skjemaType ?? null}
+        />
+      )}
+      {motebehovArbeidsgiver && (
+        <MotebehovArbeidsgiverKvittering
+          motebehov={motebehovArbeidsgiver}
+          skjemaType={motebehovArbeidsgiver?.skjemaType ?? null}
+        />
+      )}
+    </div>
+  ) : (
+    <BodyShort size="small">
+      Alle tidligere møtebehov er behandlet, se møtebehovhistorikken for flere
+      detaljer.
+    </BodyShort>
+  );
+}
+
+export default function MotebehovKvittering() {
+  const { data: motebehov } = useMotebehovQuery();
+  const { latestOppfolgingstilfelle } = useOppfolgingstilfellePersonQuery();
+
+  const sortertMotebehov = motebehov.sort(sorterMotebehovDataEtterDato);
+  const motebehovInActiveTilfelle = getMotebehovInActiveTilfelle(
+    sortertMotebehov,
+    latestOppfolgingstilfelle
+  );
+  const harMotebehovInTilfelle = !!(
+    motebehovInActiveTilfelle.length > 0 && latestOppfolgingstilfelle
+  );
+
+  return sortertMotebehov.length === 0 ? (
+    <BodyShort>Ingen tidligere møtebehov</BodyShort>
+  ) : harMotebehovInTilfelle ? (
+    <MotebehovInCurrentTilfelle
+      motebehovInTilfelle={motebehovInActiveTilfelle}
+      oppfolgingstilfelle={latestOppfolgingstilfelle}
+    />
+  ) : (
+    <UbehandledeMotebehovUtenforTilfelle
+      sorterteMotebehovUtenforTilfelle={sortertMotebehov}
+    />
```

Comment:
Disse komponentene blir ikke perfekte fordi de isolert sett kan gi feiltilstander. Men i denne konteksten så klarer vi å gjøre at ting blir "riktig", og vi har kjørt på med så beskrivende navn som mulig på komponent og props for at det skal gi mening. Komponentene er lagt inn i denne filen uten å eksporteres nettopp for å ikke eksponere de ut mot omverdenen.

Altså: `sorterMotebehov` som sendes inn på L335 kan i teorien være møtebehov innenfor et tilfelle, men det er en umulighet når den brukes i if-elsen slik som nå.